static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,77 @@
+/* global SETTINGS: false */
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Jumbotron from '../components/Jumbotron';
+import PrivacyForm from '../components/PrivacyForm';
+import ProfileFormContainer from './ProfileFormContainer'
+import {
+  saveProfilePrivacy,
+  updateProfilePrivacy,
+} from '../util/profile_edit';
+import {
+  startProfileEdit,
+} from '../actions/index';
+
+class SettingsPage extends React.Component {
+
+  static propTypes = {
+    dispatch:       React.PropTypes.func.isRequired,
+    userProfile:    React.PropTypes.object.isRequired,
+    ui:             React.PropTypes.object.isRequired,
+    errors:         React.PropTypes.object
+  };
+
+  constructor(props) {
+    super(props);
+    const { dispatch } = this.props;
+    const username = SETTINGS.username;
+    dispatch(startProfileEdit(username));
```

Comment:
I think it might be better to do this in a `componentWillMount` callback instead of a constructor